### PR TITLE
Added first-chatbot documentation

### PR DESCRIPTION
**Pull request by daehli**

### CODE REVIEW

File: docs/overview/cloud/first-chatbot.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+---
+
+--------------------
+
+# Introduction
+
+The botpress cloud is out. This version is a little bit different from the open source version. The beta version will be focusing on the deployment part. In this version we have build a conveniant way to deploy a chatbot. The Frontend is the same but with extra feature. You can use the atom version or the latest docker release. 
+
+This new product is better in lot of way. It solve the time-to-production, in one click, you have a deployed chatbot that can work on various platform. It also solve, the collaboration between conversation designer. It come with all the tool for post-production (manage and improvement). Enought of talking, let's jump into the lesson. 
+
```
Comment (ptrckbp) [inline docs/overview/cloud/first-chatbot.mdx:14]: #Instructions

File: docs/overview/cloud/first-chatbot.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+2. Create an account > Click **Sign Up**
+
+![Sign Up](/assets/cloud/login.png)
+
+3. **Sign In** with your account
+
+## Download the binary
+
+Download the binary
+
```
Comment (ptrckbp) [inline docs/overview/cloud/first-chatbot.mdx:29]: 1. Go to botpress's cloud beta
2. Click the download button for your operating system

File: docs/overview/cloud/first-chatbot.mdx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+## Create a chatbot on the cloud platform
+
+1. Click on **Select Chatbot** > **Create Chatbot**
+2. Enter the chatbot name
+![chatbot name](/assets/cloud/first-chatbot.jpg)
+3. Copy the **key** and the **secret** for the next step
+![Key from the studio](/assets/cloud/cloud-key.png)
+
+## Create a chatbot in the studio
```
Comment (ptrckbp) [inline docs/overview/cloud/first-chatbot.mdx:46]: While you download, Create a ...

File: docs/overview/cloud/first-chatbot.mdx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+:::info
+
+Do you know, you can use docker to install the botpress studio on your computer
+```
+docker run -p 3000:3000 --rm -it ghcr.io/botpress/botpress/server:next
+```
+:::
+
+## Create a chatbot on the cloud platform
+
```
Comment (ptrckbp) [inline docs/overview/cloud/first-chatbot.mdx:39]: Note / attention : You are creating a chatbot on the cloud platform and you'll create a corresponding one in the app, and link them together later.

**Review by ptrckbp — CHANGES_REQUESTED** [review]

File: docs/overview/cloud/first-chatbot.mdx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+![chatbot name](/assets/cloud/first-chatbot.jpg)
+3. Copy the **key** and the **secret** for the next step
+![Key from the studio](/assets/cloud/cloud-key.png)
+
+## Create a chatbot in the studio
+
+In the studio,  
+
+1. `Create Bot` > `New Bot` > `Enable Botpress Cloud`
+2. Copy the keys from the botpress cloud.
```
Comment (ptrckbp) [inline docs/overview/cloud/first-chatbot.mdx:51]: key and secret

File: docs/overview/cloud/first-chatbot.mdx [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+3. Create a node text.
+4. Train your chatbot
+
+:::caution
+If the bot is not train the telegram part will not work.
+:::
+
+## In the cloud platform, enable telegram
+
+1. Open the botfather on [telegram](https://t.me/botfather)
```
Comment (ptrckbp) [inline docs/overview/cloud/first-chatbot.mdx:62]: The botfather is a connector app to build chatbots on Telegram.

File: docs/overview/cloud/first-chatbot.mdx [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+2. Start the conversation
+3. Type `/newbot` and follow the instruction.
+![telegram /newbot](/assets/cloud/telegram.png)
+4. Get the HTTP API from the botfather.
```
Comment (ptrckbp) [inline docs/overview/cloud/first-chatbot.mdx:66]: HTTP API -> HTTP API token

Botfather will say something like : Use this token to access the http API

File: docs/overview/cloud/first-chatbot.mdx [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+![Key from the studio](/assets/cloud/cloud-key.png)
+
+## Create a chatbot in the studio
+
+In the studio,  
+
+1. `Create Bot` > `New Bot` > `Enable Botpress Cloud`
+2. Copy the keys from the botpress cloud.
+![Key from the studio](/assets/cloud/studio-key.png)
+3. Create a node text.
```
Comment (ptrckbp) [inline docs/overview/cloud/first-chatbot.mdx:53]: Explain that we'll use this to test the chatbot in Telegram later.

File: docs/overview/cloud/first-chatbot.mdx [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+4. Train your chatbot
```
Comment (ptrckbp) [inline docs/overview/cloud/first-chatbot.mdx:54]: Image pointing to the train chatbot button

File: docs/overview/cloud/first-chatbot.mdx (hunk 5 above)
Comment (ptrckbp) [inline docs/overview/cloud/first-chatbot.mdx:51]: Make sure the user is redirected after clicking "Create bot" there should be a step otherwise

**Review by ptrckbp — CHANGES_REQUESTED** [review]

File: docs/overview/cloud/first-chatbot.mdx [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+
+1. Go on the botpress [cloud](https://app.botpress.dev)
+
+2. Create an account > Click **Sign Up**
+
+![Sign Up](/assets/cloud/login.png)
+
+3. **Sign In** with your account
+
+## Download the studio binary
```
Comment (ptrckbp) [inline docs/overview/cloud/first-chatbot.mdx:26]: Download Botpress 12.27.0 Beta

File: docs/overview/cloud/first-chatbot.mdx [hunk 11]
```diff
@@ -0,0 +1,80 @@
+---
+id: first-chatbot
+title: First chatbot
+version: beta
+---
+
+--------------------
+
+# Introduction
+
```
[the remark this answers is not in the document]
Comment (daehli) [reply docs/overview/cloud/first-chatbot.mdx:10]: I put a place holder

**Pull request merged**